### DISCUSSION TITLE
Add meta data to MD block

### DISCUSSION THREAD

**Discussion by gual11** [danielhrisca/asammdf#765] [General]
In mdf4 format there ist also the possibility to store meta data in e tags. 
Is it possible to write new e Tags with this asammdf library?

this is an example structure:
[mdf_example.txt](https://github.com/danielhrisca/asammdf/files/9533130/mdf_example.txt)

I want to add more e tags to this, to store more metadata

**Comment by danielhrisca** [comment]
Just assign that xml string to the channel comment

```python
ch.comment = "<CNComment>..."
```

**Reply by gual11** [reply]
can you give a more detailled example?

**Reply by gual11** [reply]
and I don't want to add a comment to the channel, but to the header in the meta data block

**Reply by danielhrisca** [reply]
```python

mdf = MDF('my_file.mf4')

first_group = mdf.groups[0]
first_channel = first_group.channels[0]

first_channel.comment = """<CNcomment>
<!-- required substitute for TXBLOCK -->
<TX>Channel of great importance.</TX>
<!-- channel comment specific tags -->
<names>
<display>RPM</display>
</names>
<raster min="1" max="2">1.5</raster>
<!-- optional general tags -->
<common_properties>
<e name="blub" ci="1" desc="Blubber factor">17</e>
<tree name="user" desc="user doing the recording">
<e name="first name">John</e>
<e name="last name">Smith</e>
</tree>
</common_properties>
<extensions>
<extension ci="2">
<vendor_tag xmlns="http://vendor2/my-namespace">
<something/>
</vendor_tag>
</extension>
</extensions>
</CNcomment>"""

header = mdf.header
header.comment = """<HDcomment><TX>new header comment</TX></HDcomment>"""

mdf.save('new_comment.mf4')
mdf.close()
```

**Reply by gual11** [reply]
Thanks for the fast reply, I tested it, it goes in the right direction, but still not what I want.
Where the meta data is stored with this syntax is in the signal group, but I want to store it in the header group:
![image](https://user-images.githubusercontent.com/113166671/189863259-dab9a63e-cd9b-4403-9b9a-759d0a6b3239.png)

**Reply by gual11** [reply]
okay I found the .header parameter, I am going to try with that

**Reply by danielhrisca** [reply]
That is exactly what the example above shows
